From: Michael Froman <mfroman@mozilla.com>
Date: Wed, 5 Jul 2023 19:15:00 +0000
Subject: Bug 1841864 - upstream commit 4baea5b07f should properly check size
 of encoder_config_.simulcast_layers. r=jib

Differential Revision: https://phabricator.services.mozilla.com/D182813
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/a7179d8d75313b6c9c76a496e10d102da019ff4f
---
 video/video_stream_encoder.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/video/video_stream_encoder.cc b/video/video_stream_encoder.cc
index 552463e143..669f165635 100644
--- a/video/video_stream_encoder.cc
+++ b/video/video_stream_encoder.cc
@@ -1375,7 +1375,7 @@ void VideoStreamEncoder::ReconfigureEncoder() {
 
   bool is_svc = false;
   bool single_stream_or_non_first_inactive = true;
-  for (size_t i = 1; i < encoder_config_.number_of_streams; ++i) {
+  for (size_t i = 1; i < encoder_config_.simulcast_layers.size(); ++i) {
     if (encoder_config_.simulcast_layers[i].active) {
       single_stream_or_non_first_inactive = false;
       break;
